src/no/capraconsulting/buildtools/lifligcdkpipelines/LifligCdkPipelines.groovy

```diff
@@ -286,14 +286,40 @@ def configureVariables(Map config) {
 }
 
 def triggerPipeline(Map config) {
+
   def artifactsBucketName = require(config, "artifactsBucketName")
   def pipelineName = require(config, "pipelineName")
 
-  // Upload a blank file to trigger the event.
+  def githubUrl = sh(script: 'git config --get remote.origin.url', returnStdout: true).trim()
```

Comment:
(We're making some assumptions here about the name of the remote, as well as the format of the remote URL. Should be fine for our use-case, but worth noting that this likely does not generalize that well.)